packages/sdk/src/evm/core/classes/contract-wrapper.ts

```diff
@@ -510,6 +513,23 @@ export class ContractWrapper<
     // Parse the revert reason from the error
     const reason = parseRevertReason(error);
 
+    // Get contract sources for stack trace
+    let sources: ContractSource[] | undefined = undefined;
+    const storage = new ThirdwebStorage(); // Take thirdweb storage in constructor at some point?
```

Comment:
Do we not have one available in this class? Could at least reuse the instance instead of re-creating one everytime